tests/src/common.rs

```diff
@@ -0,0 +1,61 @@
+use serde:: {Serialize, Deserialize};
+use anyhow::Result;
+use rand::{distributions::Alphanumeric, Rng};
+
+pub async fn send_get(
+    url: &str
+) -> Result<reqwest::Response, reqwest::Error> {
+    let client  = reqwest::Client::new();
+    let response = client
+    .get(url)
+    .header("Content-Type", "application/json")
```

Comment:
Use predefined header strings e.g. http::header::CONTENT_TYPE and HeaderValue::from_static("application/json")
Reference: https://docs.rs/http/latest/http/

Reply:
was not aware of it!!! Thanks.